vendors/materialize/css/materialize.css

```diff
@@ -5581,7 +5581,8 @@ button.btn-floating {
   border-radius: 16px;
   background-color: #e4e4e4;
   margin-bottom: 5px;
-  margin-right: 5px;
+  margin-right: -40px;
```

Comment:
Please never modify a vendor supplied file.

Find another method to edit our own style.

Possibly, you might want to see coalaCSS repo for this kind of issue.